test/functional/apps/dashboard/group1/dashboard_unsaved_state.ts

```diff
@@ -140,6 +139,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.visualize.gotoVisualizationLandingPage();
         await PageObjects.header.waitUntilLoadingHasFinished();
         await PageObjects.dashboard.navigateToApp();
+        if (await PageObjects.dashboard.onDashboardLandingPage()) {
+          await testSubjects.existOrFail('unsavedDashboardsCallout');
+        }
```

Comment:
Adding this so that, if this failure happens again, it will be clearer **why** the failure is happening; because the failure (and the corresponding failure screenshot) happened on the dashboard itself, it took me a little bit of time to confirm that the failure was happening due to the session storage being cleared. Failing on the listing page makes this clearer.